File: lang/codeGen/fileGen/teste1.cpp
    #include <iostream>


    int divMod_retorno_00(int n, int q);
    int divMod_retorno_01(int n, int q);

    int main() {
        int n;
        int q;
        int quo;
        int res;
        n = 13;
        q = 5;
        quo = divMod_retorno_00(n, q);
        res = divMod_retorno_01(n, q);
        std::cout << 'Q';
        std::cout << ':';
        std::cout << quo;
        std::cout << '\n';
        std::cout << 'R';
        std::cout << ':';
        std::cout << res;
        std::cout << '\n';
        return 0;
    }

    int divMod_retorno_00(int n, int q) {
        return (n / q);
    }

    int divMod_retorno_01(int n, int q) {
        return (n % q);
    }

File: lang/codeGen/fileGen/teste1.java
import java.util.Scanner;

public class teste1 {
    public Scanner __Scanner = new Scanner(System.in); // Scanner para fazer a leitura de entrada pelo teclado


    int main() {
        Integer n;
        Integer q;
        Integer quo;
        Integer res;
        n = 13;
        q = 5;
        quo = divMod_retorno_00(n, q);
        res = divMod_retorno_01(n, q);
        System.out.print('Q');
        System.out.print(':');
        System.out.print(quo);
        System.out.print('\n');
        System.out.print('R');
        System.out.print(':');
        System.out.print(res);
        System.out.print('\n');
        return 0;
    }

    Integer divMod_retorno_00(Integer n, Integer q) {
        return (n / q);
    }

    Integer divMod_retorno_01(Integer n, Integer q) {
        return (n % q);
    }

    public static void main(String args[]) {
        teste1 m = new teste1();
        m.main();
    }
}